docs/packages/networking.rst

```diff
@@ -9,6 +9,7 @@ Networking
 
  * `Autobahn-cpp <https://github.com/ruslo/hunter/wiki/pkg.autobahn-cpp>`_ -  open-source implementations of the The WebSocket Protocol and The Web Application Messaging Protocol (WAMP>`_ network protocols.
  * `Avahi <https://github.com/ruslo/hunter/wiki/pkg.avahi>`_ - Service Discovery for Linux using mDNS/DNS-SD -- compatible with Bonjour
+ * `Beast <https://github.com/ruslo/hunter/wiki/pkg.beast>`__ - HTTP and WebSocket built on Boost.Asio in C++11
```

Comment:
Please add `Asio` to the `.. spelling::` section at the top.

* https://travis-ci.org/ruslo/hunter/jobs/248117594#L1334